src/server/__tests__/server.test.js

```diff
@@ -75,13 +76,14 @@ it('retrieve the model for a game', async () => {
   expect(response.body.credentials.length).toBe(players);
 
   // retrieve the model
-  response = await request(publicApiServer.callback()).get(`/model/${response.body.game}`);
+  response = await request(publicApiServer.callback())
+    .get(`/game/${response.body.game}/model`)
+    .auth('0', response.body.credentials[0]);
   expect(response.body).toStrictEqual(model)
 });
 
 it('download the final model for a game', async () => {
```

Comment:
Could you please also add tests that the download etc. does NOT work for wrong credentials?

Reply:
and we should probably test all possibilities:
* wrong password
* wrong username
* not Authorization header
* malformed Authorization header